index.js

```diff
@@ -1,33 +1,68 @@
 var visit = require('unist-util-visit');
 
 function sentenceNewline(ast, file, preferred, done) {
-  visit(ast, 'text', function (node) {
-    var lastDotFoundIdx = -1;
+  var blacklist = [];
+  if (typeof preferred === 'object' && !('length' in preferred)) {
```

Comment:
Hey @dirk-thomas, this `if` statement is causing a non-100% branch coverage, which I'm trying to fix. Do you know where `preferred` is documented? Or would you be interested in submitting a PR that brings up the branch coverage to 100%? I really appreciate your help.